.circleci/config.yml

```diff
@@ -52,7 +52,13 @@ step_setup_slither: &step_setup_slither
         pip install solc-select
         solc-select install 0.8.25
         solc-select use 0.8.25
-
+step_install_nvm: &step_install_nvm
```

Comment:
I wouldn't be opposed to adding a `step_install_lsof` as well, since we do it frequently